### ISSUE TITLE
How are RGB bands of drone images different from the ones of satellite images?

### ISSUE THREAD

**Issue by Genki007** [googleearthengine/geoforgoodsummit2020dronebird#3]
Can we use RGB band data taken by drone camera by the same method we analyze Landsat 8 OLI RGB? Are there any limitation in RGB band data taken by drone camera?

**Comment by mapconcierge** [comment]
Yes, general optical camera can use like the RGB sensors. But sensor sensitivity is little different. You should compare both specifications.

**Comment by Genki007** [comment]
> 
> 
> Yes, general optical camera can use like the RGB sensors. But sensor sensitivity is little different. You should compare both specifications.
Thank you very much!
We are studying seagrass habitat using Landsat 8 OLI and Sentinel 2 MSI.
We will try to obtain the drone camera data on the same day when the satellite sensor pass our region of interest.
